central/compliance/manager/run.go

```diff
@@ -159,13 +154,3 @@ func (r *runInstance) ToProto() *v1.ComplianceRun {
 	}
 	return proto
 }
-
-func (r *runInstance) shouldDelete() bool {
```

Comment:
Should this be used?

Reply:
It was removed after the only call was removed https://github.com/stackrox/stackrox/pull/8783/files/005bcb5619ec5f22f9369e9f2be57792535f4882#r1406507675

Reply:
The comment about is about the fact that was to be used with compliance schedules that were never released